### ISSUE TITLE
Issues with Kachery

### ISSUE THREAD

**Issue by MaitreyiJ** [flatironinstitute/kachery#5]
Hi!
I have been attempting to use spike forest for some spike sorting benchmarking .
I was able to clone the repo and I also downloaded the dependency Kachery-but I am having the following error when I attempt to run one of the examples in this repo(running spykingCircus sorter) 

with ka.config(fr='default_readonly'):   (Error in this line)
AttributeError: module 'kachery' has no attribute 'config'

I have tried to set the environment variable -kachery storage to some folder for temp files storage and I am not too sure how to set the password for the environment variable kachery_password.

I also attempted to run the test files in the kachery repo and I get the error-Permission denied.

I am suspecting that there is some issue with the kachery dependency.
Any help/instructions will be highly helpful.

Regards
Maitreyi

**Comment by magland** [comment]
Hi Maitreyi. Could you please confirm that you are using the right version of kachery? Did you install using `pip install kachery`? Try `pip list` to see which version is installed.

Then try a basic:
```
with ka.config(fr='default_readonly'):
    pass
```

**Comment by MaitreyiJ** [comment]
Yes I tried running the command, this time it doesn't fail, but gives a warning
Warning: unable to load preset configurations from: https://raw.githubusercontent.com/flatironinstitute/kachery/config/config/config_2019a.json

**Comment by magland** [comment]
Okay, we will look into this tomorrow, thanks for the report.

**Comment by MaitreyiJ** [comment]
Hi !
I have been able to run the sorter now-successfully (given in the example) but there is a specific directory structure I had to follow-in terms of placing the dependencies.(kachery files)

I am currently trying to read the output files generated by the example( the Jason files) with the help of widgets/ sh links.

Thanks for the support.

**Comment by magland** [comment]
What do you mean by widgets/ sh links?
Do you need help with this?

**Comment by MaitreyiJ** [comment]
Yes, I think you can help me out with this.
The following screenshot was taken after running the python script  example_spykingcircus.py
Running the script opens the command prompt and suuccefully runs the sorter (spykingCircus) on the data-set. However the command prompt closes immediately  once the spike sorting is complete and the following output is recorded in the python shell.

I am unable to understand how I should go about viewing the output files. As per the documentation , a single Jason file will contain the output . But I cant locate that file or any putput produced by the sorter.


![OutputDebuggingIssues](https://user-images.githubusercontent.com/59170109/81983061-bfc8dd80-9600-11ea-947e-b88934e55e74.png)



This is what I see in the shell(in case the screenshot doesn't load properly)
===== Hither: finished [spykingcircus] with status finished after 117.37312245368958 sec
Warning: problem writing .record.json file: C:\Users\Maitreyi\CareerDevelopmentoversumeer\SummerResearch2020\spikeforest2\.kachery-storage\sha1\e5\7d\75\e57d75df9b97b4c275976cbdaa43f101da393c99.record.json
Warning: problem writing .hints.json file: C:\Users\Maitreyi\CareerDevelopmentoversumeer\SummerResearch2020\spikeforest2\.kachery-storage\sha1\41\31\da\4131da62e3189a45b3844e5d699c96e492034905.hints.json
Warning: problem writing .record.json file: C:\Users\Maitreyi\CareerDevelopmentoversumeer\SummerResearch2020\spikeforest2\.kachery-storage\sha1\e5\7d\75\e57d75df9b97b4c275976cbdaa43f101da393c99.record.json
Warning: problem writing .hints.json file: C:\Users\Maitreyi\CareerDevelopmentoversumeer\SummerResearch2020\spikeforest2\.kachery-storage\sha1\41\31\da\4131da62e3189a45b3844e5d699c96e492034905.hints.json
hither.File(C:\Users\Maitreyi\CareerDevelopmentoversumeer\SummerResearch2020\spikeforest2\.kachery-storage\sha1\41\31\da\4131da62e3189a45b3844e5d699c96e492034905)

**Comment by magland** [comment]
It appears that you are on windows. We haven't tested anything on windows.
One thing you could do is to make a virtual linux machine.
